bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
In any case, `yarn jest` does not seem to work even if yarn is installed globally.